### ISSUE TITLE
Call with Lucy, DonDiego & Loreena

### ISSUE THREAD

**Issue by Felix-at-Swarm** [Felix-at-Swarm/felix-cardano-polkadot#65]
Meeting summary applies [The Chatham House Rule](https://en.wikipedia.org/wiki/Chatham_House_Rule)
"Under the Chatham House Rule, anyone who comes to a meeting is free to use information from the discussion, but is not allowed to reveal who made any particular comment"

Summary

Good process on the Manifesto, seems the process used to ideate and define the Manifesto was effective and arrived to engage quite a number of polkadot community members to contribute to the Manifesto's definement.

Now, focus on submitting the Manifesto on root tracks.  Then to work on the respective medium-spender referenda to launch the new Fellowship which was shared as a V1 at the Polkadot Forum.

Some challenges i personally see at the current medium-spender V1 draft.

1) KPIs/OKRs
- The current KPIs outlined in the V1 Draft are very ambitious, e.g getting 150 active Ambassadors in 4 months seems more risky than realistic. Similar to the expected slots and specialist verticals seems far too high.
- The given time to spin up the program, through a duration of 4 months may not be enough time to ensure the expected results.

Recommendation:
- Refine the KPIs/OKRs to a more realistic and achievable scope. 

2) Budget
- requesting more than 500K USD for the setup of the new fellowship once again, is rather risky than realistic, especially on the allocations to the interim Optimistic Fund

Recommendation:
- similar to the KPIs/OKRs, define a more realistic and achievable scope. Reduce the funding amount and reduce the risk of wasted funds through failure, also reduces the financial pressure and expectations.

Summing up.
The creation of the Manifesto was great and enabled a strong momentum, however, one should not get rushed by momentum, "building in the speed of trust" highly applies to the ambassadors future. Scale down on budgets as well as KPIs/OKRs for the interim and continue the same methodology as applied to the creation of the manifesto. Public calls to discuss and review the referenda drafts, engage the community, makes sure to keep building momentum, not getting rushed away.
